app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> bool:
+    parts = command.split()
+
+    source_file = parts[1]
+    destination_file = parts[2]
```

Comment:
Tru to avoid retrieving elements by index and use unpacking instead
```suggestion
    command_name, source_file, destination_file, *_ = command.split()
```